mesonbuild/interpreter.py

```diff
@@ -3467,15 +3453,21 @@ def func_configure_file(self, node, args, kwargs):
             if inputfile is not None:
                 os.makedirs(os.path.join(self.environment.build_dir, self.subdir), exist_ok=True)
                 file_encoding = kwargs.setdefault('encoding', 'utf-8')
-                missing_variables = mesonlib.do_conf_file(ifile_abs, ofile_abs,
-                                                          conf.held_object, fmt,
-                                                          file_encoding)
+                missing_variables, confdata_useless = \
+                        mesonlib.do_conf_file(ifile_abs, ofile_abs, conf.held_object,
```

Comment:
__[Flake8]__

[E127] continuation line over-indented for visual indent


<!-- Comment by [Sider](https://sider.review) -->